libraries/Microsoft.Bot.Builder.LUIS/LuisRecognizer.cs

```diff
@@ -22,24 +23,30 @@ public class LuisRecognizer : IRecognizer
         private readonly ILuisOptions _luisOptions;
         private readonly ILuisRecognizerOptions _luisRecognizerOptions;
 
+        private readonly ILuisOptions _defaultLuisOptions = new LuisRequest();
+        private readonly ILuisRecognizerOptions _defaultRecognizerOptions = new LuisRecognizerOptions {Verbose = true};
+
```

Comment:
Why have both a _luisOptions and a _defaultLuisOptions? 

Just initialize:
```
_luisOptions = new LuisRequest();
_luisRecognizerOptions = new LuisRecognizerOptions {Verbose = true};
_luisOptions = new LuisRequest();
_luisRecognizerOptions = new LuisRecognizerOptions {Verbose = true};
```
... and call it done.

Reply:
I already refactored this but didn't push, did something different but similar